common/src/main/proto/TransportMessages.proto

```diff
@@ -625,6 +636,7 @@ message PbOpenStream {
   int32 endIndex = 4;
   int32 initialCredit = 5;
   bool readLocalShuffle = 6;
+  bool shuffleDataNeedSort = 7;
```

Comment:
IIUC if this PR's optimization is enabled, `endIndex` will be Int.MAX_VALUE, so `shuffleDataNeedSort` is unnecessary.

Reply:
Fixed, thanks